stdlib/public/runtime/CMakeLists.txt

```diff
@@ -122,11 +123,15 @@ if(SWIFT_BUILD_STATIC_STDLIB AND "${sdk}" STREQUAL "LINUX")
       DEPENDS
          ${FragileSupportLibrary})
     swift_install_in_component(stdlib
-      FILES $<TARGET_FILE:${FragileSupportLibrary}>
-      DESTINATION "lib/swift_static/${lowercase_sdk}/${arch}")
+                               FILES
+                                 $<TARGET_FILE:${FragileSupportLibrary}>
+                               DESTINATION
+                                 "lib/swift_static/${LIBRARY_SUBDIR}")
   endforeach()
 
-  set(FragileSupportLibraryPrimary swiftImageInspectionShared-${SWIFT_SDK_${sdk}_LIB_SUBDIR}-${SWIFT_PRIMARY_VARIANT_ARCH})
+  string(TOLOWER "${sdk}" lowercase_sdk)
```

Comment:
I might be wrong, but isn’t this `lowercase_sdk` supposed to be `SWIFT_SDK_${sdk}_LIB_SUBDIR` instead? It is used below as in `LibraryLocationPrimary` or `DESTINATION "lib/swift_static/${lowercase_sdk}`. Notice that you already replaced a usage above for `compute_library_subdir` which uses `_LIB_SUBDIR`.